src/page/Home/Home.jsx

```diff
@@ -1,7 +1,28 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { QuizCard } from "../../component/QuizCard/QuizCard"
+
+const getCategories = async (setCategories) => {
+    const { data : {categories} } = await axios.get("api/category/");
+    setCategories(categories)
```

Comment:
Please do inside try catch, It can catch uncaught errors and display the errors in UI if needed.